public/app/features/storage/FolderDashboard.tsx

```diff
@@ -0,0 +1,79 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import { useAsync } from 'react-use';
+
+import { DataFrame, GrafanaTheme2 } from '@grafana/data';
+import { Card, Icon, Spinner, useStyles2 } from '@grafana/ui';
+
+import { DashboardModel } from '../dashboard/state';
+
+import { getGrafanaStorage } from './storage';
+
+interface Props {
+  dash: DashboardModel;
+}
+
+// This is a helper page that lists links to items in the same folder
+export function FolderDashboard({ dash }: Props) {
```

Comment:
Do we need this FolderDashboard thing? Not sure what it brings here. I suspect we will have two ways to explore / list dashboards (beyond search). A dedicated "Dashboards" browse / list with specific dashboard cards. And a generic storage admin view (similar to StoragePage that maybe has more admin actions and shows every type of entity/file type). 

Not sure where the FolderDashboard view fits here that is not solved by a Dashboard / browse / folder page? 

At least for now, I think we can skip this and revisit it later (to reduce scope and reduce duplication/purpose of views)

Reply:
Maybe generic storage browse cards and dashboard browse views can be the same the dashboard cards are pretty generic (unlike Alert rules, reports etc that have very speciifc info)